specifications/transfer-claim/transfer-claim.yml

```diff
@@ -803,7 +812,35 @@ components:
                     description: Percentage to finance of the claim total amount
                     minimum: 0
                     example: 90
-
+              claimStatus:
+                type: string
+                description: Status of account
```

Comment:
Here again, we cannot use "account" since it has a different meaning in the APIs ;) 
Here you can just say, status of the claim ;)